recipes/gmpy2/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "2.0.7" %}
+
+package:
+  name: gmpy2
+  version: {{ version }}
+
+source:
+  fn: gmpy2-{{version}}.zip
+  url: https://pypi.io/packages/source/g/gmpy2/gmpy2-{{ version }}.zip
+  md5: b5aada3ee5afb316ea94604f45192054
+
+build:
+  number: 0
+  skip: true        # [win and not py35]
+  features:
+    - vc14          # [win and py35]
+  script:
+    - python setup.py install --shared=$PREFIX                                          # [unix]
+    - python setup.py build_ext -f --shared=%LIBRARY_PREFIX% -DMSC_USE_DLL=1 install    # [win]
```

Comment:
For Windows, when using a function from a dll you have to explicitly mark it `__declspec(dllimport)` in the header. This macro would turn on that feature. Otherwise it defaults to nothing, which means that the library implementing the function should be linked statically.